backend/service/k8s/clientset.go

```diff
@@ -95,6 +103,20 @@ func newClientsetManager(rules *clientcmd.ClientConfigLoadingRules, logger *zap.
 	return &managerImpl{clientsets: lookup}, nil
 }
 
+func applyRestClientConfig(restConfig *restclient.Config, restClientConfig k8sconfigv1.RestClientConfig) {
+	if restClientConfig.Burst != 0 {
+		restConfig.Burst = int(restClientConfig.Burst)
+	}
+
+	if restClientConfig.Qps >= 0 {
+		restConfig.QPS = restClientConfig.Qps
+	}
+
+	if restClientConfig.Timeout != nil {
+		restConfig.Timeout = time.Duration(restClientConfig.Timeout.Seconds) * time.Second
```

Comment:
use `ptypes.Duration` to convert from restClientConfig.Timeout

Reply:
ah much nicer